### DISCUSSION TITLE
Include Self-RAG again

### DISCUSSION THREAD

**Discussion by AlphaMoury** [infiniflow/ragflow#2911] [Q&A]
Why did you remove Self-RAG feature from the new RagFlow version? I experienced improved results when building Chatbots.

Is there any reason for it? Are you planning to replace it with another filtering technique?

**Comment by KevinHuSh** [comment]
It's not reliable for LLM to judge relevance between the retrieved content and user's question.
And it makes the searching process much slower. 
The cons outweigh pros.

**Reply by AlphaMoury** [reply]
Could you include it as an option such that the user can decide whether to use it or not? It will be appreciated :)

**Reply by KevinHuSh** [reply]
Let's figure it out how to promote it before exposing it.
